packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -325,10 +255,9 @@ function Session(props: PropsWithChildren<Props>) {
           tdpClient.nuke();
         }}
         userHost={`${username}@${hostname}`}
-        clipboardSharingEnabled={clipboardSharingActive}
```

Comment:
I'm not sure anything is passing this into `TopBar` so it can maybe be removed from `TopBar` as well.

Reply:
https://github.com/gravitational/webapps/pull/1333#discussion_r1022069855